eclipse/sw360
# feat(ui/rest): Update project specific vulnerability action

Signed-off-by: Smruti Prakash Sahoo <smruti.sahoo@siemens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*#1300*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: Closes: #1300 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?

* Need to test updating vulnerabilities with action field from both REST(Patch - `/projects/{projectId}/vulnerabilities` ) and UI(Project detail page -> Vulnerabilities -> Select Vulnerabilities to be updated -> Change).

> Have you implemented any additional tests? - No

### Checklist
Must:
